sage-ios/SAGE/SAGE/LoginOperations.swift

```diff
@@ -0,0 +1,175 @@
+            }
+            if let email = user.email {
+                existingUser.email = email
+            }
+            if let school = user.school {
+                existingUser.school = school
+            }
+            if User.VolunteerLevel.Default != user.level {
+                existingUser.level = user.level
+            }
+            if User.UserRole.Default != user.role {
+                existingUser.role = user.role
+            }
+            if User.DefaultValues.DefaultHours.rawValue != user.totalHours {
+                existingUser.totalHours = user.totalHours
+            }
+            existingUser.verified = user.verified
+            KeychainWrapper.setObject(existingUser, forKey: KeychainConstants.kUser)
+        } else {
+            KeychainWrapper.setObject(user, forKey: KeychainConstants.kUser)
+            if let auth = authToken {
+                KeychainWrapper.setObject(auth, forKey: KeychainConstants.kAuthToken)
+            }
+        }
+    }
+    
+    static func createUser(firstName: String, lastName: String, email: String, password: String, school: Int, hours: Int, role: Int, photoData: String, completion: ((Bool) -> Void)) {
+        
+        let operationManager = AFHTTPRequestOperationManager()
+        operationManager.requestSerializer = AFJSONRequestSerializer()
+        operationManager.responseSerializer = AFJSONResponseSerializer()
+        
+        operationManager.requestSerializer.setValue("application/json", forHTTPHeaderField: "Accept")
+        operationManager.requestSerializer.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        
+        let params = ["user":
+            [
+                "first_name": firstName,
+                "last_name": lastName,
+                "email": email,
+                "password": password,
+                "role": role,
+                "volunteer_type": hours,
+                "data": photoData,
+                "school_id": school // for now
+            ]
+        ]
+        
+        operationManager.POST(StringConstants.kEndpointCreateUser, parameters: params, success: { (operation, data) -> Void in
+            let userDict = data["session"]!!["user"]!!
+            let user = User(propertyDictionary: userDict as! [String: AnyObject])
+            let authToken = data["session"]!![UserConstants.kAuthToken] as! String
+            let schoolDictionary = data["session"]!!["school"]!!
+            let school = School(propertyDictionary: schoolDictionary as! [String : AnyObject])
+            user.school = school
+            
+            LoginOperations.storeUserDataInKeychain(user, authToken: authToken)
+            completion(true)
+            
+            }) { (operation, error) -> Void in
+                completion(false)
+        }
+    }
+    
+    static func loginWith(email: String, authToken: String? = nil, password: String? = nil, completion: ((Bool) -> Void)) {
```

Comment:
can you make a comment here to say that even if the user is logged in, this also checks if a user is verified?